### ISSUE TITLE
Add ability to set default units

### ISSUE THREAD

**Issue by bryanwweber** [bryanwweber/thermostate#28]
It would be very useful to be able to set the default units for a particular problem. For instance:

```python
from thermostate import State, Q_, set_default_units
set_default_units("SI")
st = State("water", p=Q_(101325, "Pa"), T=Q_(25.0, "degC"))
print(st.T)
```
would give `25.0 degC` rather than right now, it would give `298.15 K`. This would save having to convert all the calculated properties, especially more complicated ones like `s`.

One disadvantage would be how to manage this global state in the program. Should the default units be a module level global (this may lead to inconsistent results, in the sense that changing it in the middle of a problem will be weird)? Should it be able to be changed only on each instance (this doesn't seem that useful, since you'd have to set it for each instance when its created)?

**Comment by mdbernard** [comment]
Problems are typically done in their own notebook, and in one set of units. If that's by far the most common use case, it would make sense to make it module-level rather than set it for each State instance, to take workload off the users.

For PowerUsers(TM) It'd be nice to have both features. For example, you could use
```
thermostate.set_default_units('SI')
```
to set the module-level default and
```
st = State("water", p=Q_(101325, "Pa"), T=Q_(25.0, "degC"), [units='SI'])
```
to optionally overwrite at the instance-level, but I see the module-level one being more useful.

**Comment by bryanwweber** [comment]
Thanks @mdbernard that's a good idea. Managing the global/module level state will be the tricky bit I think.

**Comment by bryanwweber** [comment]
Actually, now that I think about it, I'm pretty sure that Pint has some ability to do this. That might be a good approach for the global level. Then there could be an instance variable on `State` that defaults to `None`. That variable would be checked when each property is computed, if not equal to `None` the units of the property are changed appropriately. This wouldn't be too inefficient because the properties are only computed once when the state is set.

**Comment by bryanwweber** [comment]
https://pint.readthedocs.io/en/stable/systems.html

**Comment by bryanwweber** [comment]
Closed by #35
